junit5/src/main/java/sunstone/core/TemplateCloudDeploymentManager.java

```diff
@@ -1,13 +0,0 @@
-package sunstone.core;
-
-import java.util.Map;
-
-/**
- * Manager of cloud resources. Handles deploy operation, registering resources and deleting them. This is meant to be used
- * for deploying parametrized templates. Parameters are supposed to be defined in an annotation as a String.
- *
- * Used by {@link SunstoneCloudDeploy}.
- */
-interface TemplateCloudDeploymentManager extends CloudDeploymentRegistry {
```

Comment:
why did you changed your mind and removed `TemplateCloudDeploymentManager extends CloudDeploymentRegistry` ?

Reply:
There were two reasons. They differ too much and have less in common - I need to deploy to specific group and region for Azure and I need to deploy to specific region for AWS. I need to also provide clients that are different.

Second reason that follows the first one, I need to work directly with manager classes due to its differences and interface is no longer being used - it has become obsolete.